// [[Reason: We made this is an error, thus the auto-fixer is trying to fix it. Although it's possible syntactically to figure out that E is an enum, I think we should rather make this an error for now.]] ////

//// [tests/cases/compiler/ambientConstLiterals.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -15,9 +15,9 @@
 declare const c5: 123;
 declare const c6: -123;
 declare const c7 = true;
 declare const c8: E.A;
-declare const c8b = E["non identifier"];
+declare const c8b: (typeof E)["non identifier"];
 declare const c9: {
     x: string;
 };
 declare const c10: number[];
